app/views/dashboard/_subscription_status_widget.html.erb

```diff
@@ -3,7 +3,7 @@
 </h4>
 
 <% organizations = Organization.current.present? ? [Organization.current] : User.current.allowed_organizations %>
-<% subscriptions = organizations.collect { |org| org.redhat_provider.index_subscriptions }.flatten %>
+<% subscriptions = organizations.collect { |org| Katello::Pool.joins(:subscription).where("#{Katello::Subscription.table_name}.organization_id" => org.id) }.flatten %>
```

Comment:
probably, I'll take a look.

If it gets complicated we can revisit closer to end of day today, I would like this merged in the next day or two if possible.